### ISSUE TITLE
Error in  date_search with default end=None

### ISSUE THREAD

**Issue by dz0** [python-caldav/caldav#55]
```
File "caldav_demo.py", line 43, in <module>
     results = calendar.date_search( datetime(2010, 5, 1) )
File "caldav/objects.py", line 567, in date_search
    end = _fix_tz(end)
  File "caldav/objects.py", line 40, in _fix_tz
    return dt.replace(tzinfo=tzlocal.get_localzone())
AttributeError: 'NoneType' object has no attribute 'replace'
```

**Comment by tobixen** [comment]
This is a really very bad bug, I think there are tests for this ... but I admit I rushed in some "obvious bugfixes" recently without bothering to run the tests.  Oups.  I suppose I will rush out a new bugfix release soon.

This bug seems related to the last comment in #52.

I should get #46 fixed so I can have github run tests for me.

**Comment by tobixen** [comment]
I really should spend some more time researching this and other issues, but very much going on right now ... need to sleep.  Could you check if the last commit solves your issue?

**Comment by Achimh3011** [comment]
Should be fixed with #59.

**Comment by tobixen** [comment]
Closing this issue
